dev-docs/publisher-api-reference/setConfig.md

```diff
@@ -1404,6 +1404,63 @@ pbjs.setConfig({
 
 <a name="setConfig-Generic-Configuration"></a>
 
+### Enable Previous Auction Info
+
+The Previous Auction Info module enables functionality to collect prior auction data for participating bidders and publishers (This feature is `opt-in`, by default it is disabled).  The following requirements must be in place for the module to work:
+
+* Publisher's who would like to participate must explicitly configure PBJS to do so:
+
+```javascript
+pbjs.setConfig({previousAuctionInfo: true})
+```
+
+* Only bid adapters who have enabled this feature will be able to utilize it.  To do this, a bid adapter must add the following to their bidAdapter.js file:
+
+```javascript
+import { enablePreviousAuctionInfo } from '../libraries/previousAuctionInfo/previousAuctionInfo.js';
+enablePreviousAuctionInfo({ bidderCode: 'some bidder code' });
```

Comment:
see https://github.com/prebid/Prebid.js/issues/12822 - this approach is not friendly to PBS and we may want to change it.